python-sdk/nuimages/nuimages.py

```diff
@@ -701,52 +797,58 @@ def render_image(self,
         im = Image.open(im_path)
 
         # Initialize drawing.
-        font = ImageFont.load_default()
+        font = get_font(fonts_valid=['FreeSerif.ttf', 'FreeSans.ttf', 'Century.ttf', 'Calibri.ttf', 'arial.ttf'],
```

Comment:
For better reusability, can you move the default fonts into the get_font method, rather than listing them here?